docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.3"
+
+services:
+  database:
+    image: mysql
+    restart: on-failure
```

Comment:
What's `on-failure`?

Reply:
This tells Docker Compose to restart MySQL server when the server command exits with a non-zero exit code.